app/actors/sufia/file_set_actor.rb

```diff
@@ -0,0 +1,18 @@
+module Sufia
+  class FileSetActor < CurationConcerns::FileSetActor
+    # Overrides the method in CurationConcerns::FileSetActor so that if a nil work
+    # is passed, a default work is create instead.
+    def create_metadata(upload_set_id, work, file_set_params = {})
+      work ||= default_work
+      super
+    end
+
+    private
+
+      def default_work
+        GenericWork.create(title: ['Default title']) do |w|
+          w.apply_depositor_metadata(user)
```

Comment:
we will need to add upload_set_id here. Because the batch controller updates metadata on the works.